openmetadata-ui/src/main/resources/ui/src/styles/spacing.less

```diff
@@ -397,3 +397,7 @@
 .m--ml-1 {
   margin-left: -0.25rem /* -4px */;
 }
+
+.m--ml-0\.5 {
+  margin-left: -0.125rem /* -2px */;
```

Comment:
I really discourage to use negative margin or padding

Reply:
@chirag-madlani, Sure, I'm using negative margin only if really needed.